### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by tcarwash** [tcarwash/home-assistant_noaa-space-weather#31] [Announcements]
## :boom: Breaking Changes

* Fix sensors returning text value instead of numeric (#30) @tcarwash
This changes the behavior of all sensors to return a numeric value instead of a text state. May affect anyone using the integration's values in automation etc.
* Fixed devcontainer: replaced deprecated base container. Having an issue where urllib3 needs to be reinstalled before homeassistant will run, but other than that it's working much better

<hr /><em>This discussion was created from the release <a href='https://github.com/tcarwash/home-assistant_noaa-space-weather/releases/tag/untagged-37ed2d27bb4ca0028210'>v2.0.0</a>.</em>

**Comment by tcarwash** [comment]
I also saw that the one day m_class was returning the data for the one day x_class, so that is fixed in this release as well. Thanks for bearing with me, this is my first time working on anything like this.

**Comment by emefff** [comment]
Hi, 
now everything is working as advertised. I am getting graphs instead of bar graphs,
thanks,
Mario.

**Reply by tcarwash** [reply]
Perfect, glad it's working better now!
